pkg/webhook/mutation/namespace_mutator/otel.go

```diff
@@ -0,0 +1,32 @@
+package namespace_mutator
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/dtotel"
+	webhookotel "github.com/Dynatrace/dynatrace-operator/pkg/webhook/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	// the attribute key needs to be added to the allow list on the receiving tenant
+	mutatedNamespaceNameKey = "webhook.mutationrequest.namespace.name"
+)
+
+func countHandleMutationRequest(ctx context.Context, namespace string) {
+	dtotel.Count(ctx, webhookotel.Meter(), "handledPodMutationRequests", int64(1),
+		attribute.String(webhookotel.WebhookPodNameKey, webhookotel.GetWebhookPodName()),
+		attribute.String(mutatedNamespaceNameKey, namespace))
+}
+
+func spanOptions(opts ...trace.SpanStartOption) []trace.SpanStartOption {
+	options := make([]trace.SpanStartOption, 0)
+	options = append(options, opts...)
+	options = append(options, trace.WithAttributes(
+		attribute.String(webhookotel.WebhookPodNameKey, webhookotel.GetWebhookPodName()),
+
+		// TODO: this is just for showcasing now, should be removed in the future
+		attribute.String("debug.info", "foobar")))
```

Comment:
I don't see what is this showcasing
there is already an attribute, so we don't need a second one that is just noise

Reply:
It should show, that one can attach arbitrary attributes. But you might be right, it's worth is probably very limited.

Reply:
I would remove it too

Reply:
removed [97b95f6](https://github.com/Dynatrace/dynatrace-operator/pull/2289/commits/97b95f6dc9dc947241d2d154667b93effda24027)

Reply:
well I still see it 😄 

https://github.com/Dynatrace/dynatrace-operator/pull/2289/files#diff-4cf48530dbad519d66501942ecbbceccc2d0ec368545789a52b4458a7da029e1R30